pom.xml

```diff
@@ -608,11 +602,13 @@
         <groupId>org.junit.vintage</groupId>
         <artifactId>junit-vintage-engine</artifactId>
         <version>${junit.version}</version>
+        <scope>test</scope>
       </dependency>
       <dependency>
         <groupId>org.mockito</groupId>
         <artifactId>mockito-core</artifactId>
         <version>${mockito.version}</version>
+        <scope>test</scope>
```

Comment:
It's not a good idea to put the `<scope>` in the `<dependencyManagement>` section of the POM, because that will propagate, and require the `<scope>compile</scope>` to be explicitly added (normally, it's not needed, because it's the default scope). It's better to just manage the version in the `<dependencyManagement>` section, and defer adding a scope until the dependency is actually used.

It is unlikely that these dependencies are going to be used in a scope other than `test`, but the only reason to put the scope here is to omit it later when it is used. However, omitting it later will make it *appear* as though the scope is used is the `compile` scope, so it doesn't really have any benefit to adding it here.